webkitgtk-2.20: build broken on hydra

https://hydra.nixos.org/build/71902079/nixlog/1

I also got the issue when doing a nixos-rebuild on my 2-core laptop, but it succeeded on the same laptop when using nix-build, which means that all cores are available for just webkitgtk.

This is the error message I got:

```

builder for '/nix/store/5a00rqavnjk83naq1h6mgmbdgc1v1f0r-webkitgtk-2.20.0.drv' failed with exit code 1; last 10 log lines:
                   from ../Source/JavaScriptCore/heap/Heap.h:33,
                   from ../Source/JavaScriptCore/heap/DeferGC.h:29,
                   from ../Source/JavaScriptCore/runtime/ConcurrentJSLock.h:28,
                   from ../Source/JavaScriptCore/bytecode/ArrayProfile.h:28,
                   from ../Source/JavaScriptCore/llint/LLIntOffsetsExtractor.cpp:28:
  ../Source/JavaScriptCore/API/JSHeapFinalizerPrivate.h:29:41: fatal error: JavaScriptCore/JSContextRef.h: No such file or directory
   #include <JavaScriptCore/JSContextRef.h>
                                           ^
  compilation terminated.
ninja: build stopped: subcommand failed.
```

--- a/pkgs/development/libraries/webkitgtk/2.20.nix
+++ b/pkgs/development/libraries/webkitgtk/2.20.nix
@@ -39,6 +39,12 @@ stdenv.mkDerivation rec {
     patchShebangs .
   '';
 
+  postConfigure = ''
+    # A stopgap for a non-deterministic build failure when using only one core
+    # Upstream bug: https://bugs.webkit.org/show_bug.cgi?id=183788#c4
+    ninja JavaScriptCoreForwardingHeaders WTFForwardingHeaders
+  '';
+
   cmakeFlags = [
   "-DPORT=GTK"
   "-DUSE_LIBHYPHEN=0"

